packages/web-react/src/components/Toast/ToastBar.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from 'classnames';
+import React, { MutableRefObject, useRef } from 'react';
+import { Transition, TransitionStatus } from 'react-transition-group';
+import { useStyleProps } from '../../hooks';
+import { SpiritToastBarProps } from '../../types';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import { VisuallyHidden } from '../VisuallyHidden';
+import { DEFAULT_ICONS, TRANSITIONING_STYLES, TRANSITION_DURATION } from './constants';
+import { useToastBarStyleProps } from './useToastBarStyleProps';
+
+const ToastBar = (props: SpiritToastBarProps) => {
+  const {
+    id,
+    children,
+    closeLabel = 'Close',
+    color = 'inverted',
+    hasIcon,
+    iconName,
+    isDismissible,
+    isOpen = true,
+    onClose = () => {},
+    ...restProps
+  } = props;
+  const rootElementRef: MutableRefObject<HTMLDivElement | null> = useRef(null);
+  const { classProps, props: modifiedProps } = useToastBarStyleProps({
+    ...restProps,
+    color,
+    isOpen,
+    isDismissible,
+    id,
+  });
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  return (
+    <Transition in={isOpen} nodeRef={rootElementRef} timeout={TRANSITION_DURATION}>
+      {(transitionState: TransitionStatus) => (
+        <div
+          {...styleProps}
+          {...otherProps}
+          // Element implicitly has an 'any' type because expression of type 'TransitionStatus' can't be used to index type '{ entering: string; entered: string; exiting: string; exited: string; }'.
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          className={classNames(classProps.root, TRANSITIONING_STYLES[transitionState], styleProps.className)}
+          ref={rootElementRef}
+        >
+          <div className={classProps.content}>
+            {(hasIcon || iconName) && <Icon name={iconName || DEFAULT_ICONS[color]} boxSize={20} />}
```

Comment:
Please, consider adding the bog size number into the constants file.

Reply:
boxSize moved to constants